contrib/ci/fedora.sh

```diff
@@ -65,6 +69,7 @@ cp $HOME/rpmbuild/RPMS/*/*.rpm dist
 
 if [ "$CI" = "true" ]; then
 	sed "s,^DisabledPlugins=.*,DisabledPlugins=," -i /etc/fwupd/daemon.conf
+	sed "s,^AllowEmulation=false,AllowEmulation=true," -i /etc/fwupd/daemon.conf
```

Comment:
do the same thing to `contrib/debian/fwupd-tests.postinst` and `contrib/debian/tests/ci` and `contrib/debian/fwupd-tests.postrm`

Reply:
Done, thanks.